src/azure-cli/azure/cli/command_modules/storage/_params.py

```diff
@@ -742,6 +743,24 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
         c.argument('message_id', options_list='--id')
         c.argument('content', type=unicode_string, help='Message content, up to 64KB in size.')
 
+    with self.argument_context('storage remove') as c:
+        from .completers import file_path_completer
+
+        c.extra('container_name', container_name_type)
+        c.extra('blob_name', options_list=('--name', '-n'), arg_type=blob_name_type,
+                validator=validate_azcopy_remove_url)
```

Comment:
Why do you only validate azcopy remove url for blob_name? What about share name?

Reply:
Updated with validate_azcopy_remove_arguments. It will validate all the arguments.